packages/terra-i18n/src/translationLoaders.js

```diff
@@ -1,3 +1,11 @@
+const loadArTranslation = (callback, scope) => {
```

Comment:
these all seem to be the same method. Would it be simpler to just make a single function that can create these based off a variable passed in?

Something along the lines of
```
const loadTranslationGenerator = (locale => {
  return (callback, scope) => {
   require.ensure([], (require) => {
     // eslint-disable-next-line
    const i18n = require(locale.toString());
    callback.call(scope, i18n);
  }, `${locale}-translations`);
})

...

const translationLoaders = {
  en: loadTranslationGenerator('en'),
  'en-GB': loadTranslationGenerator('en-GB'),
  ....
};
```

not sure on the syntax

Reply:
i tried that but webpack was not happy:
<img width="1044" alt="screen shot 2017-09-14 at 5 02 04 pm" src="https://user-images.githubusercontent.com/5943136/30457831-9683fffe-996e-11e7-8dfb-3034d624608d.png">

Reply:
Yes, I believe we need to leave as is, webpack is picky about being able to statically analyze modules.